src/fusor/translator.py

```diff
@@ -589,10 +595,23 @@ async def from_arriba(
             )
         )
         rf = bool(arriba.rf == "in-frame") if arriba.rf != "." else None
+
+        # Process read data and fusion_transcript sequence
         read_data = ReadData(
             spanning=SpanningReads(spanningReads=arriba.discordant_mates)
         )
         contig = ContigSequence(contig=arriba.fusion_transcript)
+        linker_regex = r"\|([atcg]+)\|"
```

Comment:
I'm kind of surprised we don't have this somewhere already. Should we make it a global var?